### PR TITLE
Adding meta files

### PR DESCRIPTION
**Pull request by loevgaard**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+/.editorconfig              export-ignore
+/.gitattributes             export-ignore
+/.gitignore                 export-ignore
+/.scrutinizer.yml           export-ignore
+/.travis.yml                export-ignore
+/behat.yml.dist             export-ignore
+/easy-coding-standard.yaml  export-ignore
+/phpspec.yml.dist           export-ignore
+/phpstan.neon               export-ignore
+/README.md                  export-ignore
```
Comment (stefandoorn) [inline .gitattributes]: I would keep the README :-)

File: .gitattributes [hunk 2]
```diff
@@ -1,6 +1,14 @@
-/tests export-ignore
-/spec export-ignore
-/.travis.yml export-ignore
-/.gitignore export-ignore
-/.gitattributes export-ignore
-/phpunit.xml.dist export-ignore
```
Comment (stefandoorn) [inline .gitattributes]: Why not exclude phpunit config file?

File: .scrutinizer.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+build:
```
Comment (stefandoorn) [inline .scrutinizer.yml]: I'm in doubt about Scrutinizer, recently seen a bit too much issues on big projects with many dependencies. But we can try it of course to see what it brings us :)

File: .gitattributes (hunk 1 above)
Comment (loevgaard) [reply .gitattributes]: Why is that? :D

In my personal experience I have never, as in 0 times, read a readme in a vendor folder

File: .gitattributes (hunk 2 above)
Comment (loevgaard) [reply .gitattributes]: I probably copied the file from another plugin where no phpunit config file was present. I have added it now :)

File: .scrutinizer.yml (hunk 3 above)
Comment (loevgaard) [reply .scrutinizer.yml]: It sometimes brings issues that phpstan haven't found. Also I like the code quality badge ;)

File: .gitattributes (hunk 1 above)
Comment (stefandoorn) [reply .gitattributes]: Haha, I do that :-)

**Comment by stefandoorn** [comment]
Can you rebase against master? I've merged the skeleton upgrade PR.

File: .gitattributes (hunk 1 above)
Comment (loevgaard) [reply .gitattributes]: Crazy :D

**Pull request merged**